src/binary-reader-objdump.cc

```diff
@@ -46,9 +47,10 @@ class BinaryReaderObjdumpBase : public BinaryReaderNop {
  protected:
   const char* GetFunctionName(Index index);
 
-  ObjdumpOptions* options = nullptr;
-  const uint8_t* data = nullptr;
-  size_t size = 0;
+  ObjdumpOptions* options;
+  ObjdumpState* objdump_state_;
```

Comment:
There is already state member from the reader interface (admittedly without an _, but have state and state_ would be strange).